kubectl-fdb/cmd/remove_process_group.go

```diff
@@ -109,10 +117,12 @@ kubectl fdb -n default remove process-group -c cluster --remove-all-failed
 	return cmd
 }
 
-// replaceProcessGroups adds process groups to the removal list of their respective clusters
-// if a clusterName is specified, it will ONLY do so for the specified cluster
-func replaceProcessGroups(kubeClient client.Client, clusterName string, ids []string, namespace string, clusterLabel string, withExclusion bool, wait bool, removeAllFailed bool, useProcessGroupID bool) error {
-	if len(ids) == 0 && !removeAllFailed {
+// replaceProcessGroups adds process groups to the removal list of their respective clusters.
+// If clusterName is specified, it will ONLY do so for the specified cluster.
+// If processClass is specified, it will ignore the given ids and remove all processes in the given cluster whose pods
+// have a processClassLabel matching the processClass.
+func replaceProcessGroups(kubeClient client.Client, clusterName string, ids []string, namespace string, clusterLabel string, processClass string, withExclusion bool, wait bool, removeAllFailed bool, useProcessGroupID bool) error {
```

Comment:
would a struct of common option fields be helpful or annoying to others? The list of parameters / options is getting lengthly

Reply:
I'm fine with creating a struct that keeps all parameters.